docs/config/README.md

```diff
@@ -2445,6 +2446,78 @@ number_threshold = 4
+| `commit_id_prefix_style` | `'blue'`                                                                                                                                                                                                                                                            | The style for unique part of commit hash.                                                                                       |
+| `commit_id_suffix_style` | `'bright-black'`                                                                                                                                                                                                                                                    | The style for the rest of commit hash.                                                                                          |
+| `truncation_length`      | `8`                                                                                                                                                                                                                                                                 | Truncates the commit and change id to `truncation_length` characters.                                                           |
+| `branch_style`           | `'purple'`                                                                                                                                                                                                                                                          | The style for the branch name.                                                                                                  |
+
+### Variables
+
+```
+[🍐 rnyynlop 811ca1a4 on push-rnyynlopprno*]
+```
+
+| Variable           | Example   | Description                                                       |
+| ------------------ | --------- | ----------------------------------------------------------------- |
+| `symbol`           | `🍐`      | Mirrors the value of option `symbol`.                             |
+| `change_id_prefix` | `r`       | Unique part of the change id, styled by `change_id_prefix_style`. |
+| `change_id_suffix` | `nyynlop` | The rest of the change id, styled by `change_id_suffix_style`.    |
+| `commit_id_prefix` | `811`     | Unique part of the commit id, styled by `commit_id_prefix_style`. |
+| `commit_id_suffix` | `ca1a4`   | The rest of the commit id, styled by `commit_id_suffix_style`.    |
+| `branch`           | `master`  | The current branch name.                                          |
+
+### Example
+
+```toml
+# ~/.config/starship.toml
+
+[jj]
```

Comment:
```suggestion
[jj_status]
```

The docs here are out of sync with the code. Also needs changing in the initial prose.